### PR TITLE
Add my reviews

### PR DESCRIPTION
**Pull request by MariemBaccari**
This unexpectedly small PR adds a my reviews fragment that's similar to the feed but shows you your own reviews.

Screenshot_20230519-110417_MediaTo

### CODE REVIEW

File: app/src/main/res/drawable/ic_my_reviews.xml
Comment (MeKHell) [inline app/src/main/res/drawable/ic_my_reviews.xml:1]: What is this file ?

File: app/src/main/res/menu/bottom_menu.xml [hunk 1]
```diff
@@ -12,6 +12,10 @@
     android:id="@+id/search"
     android:title="Search"
     android:icon="@drawable/ic_search"/>
+  <item
+      android:id="@+id/my_reviews"
+      android:title="My"
```
Comment (MeKHell) [inline app/src/main/res/menu/bottom_menu.xml:17]: Would not `Mine` be better ?

**Review by MeKHell — APPROVED** [review]
Good. I commented two points that my need to be modified. But they are non-blocking ones.

File: app/src/main/res/drawable/ic_my_reviews.xml
Comment (MariemBaccari) [reply app/src/main/res/drawable/ic_my_reviews.xml:1]: it's the bottom menu icon for the new fragment !

File: app/src/main/res/menu/bottom_menu.xml (hunk 1 above)
Comment (MariemBaccari) [reply app/src/main/res/menu/bottom_menu.xml:17]: I had a lot of doubts about that and wanted to put "My Reviews" first but that was too long... I also thought about "Mine" but it looked a little bit weird in the app so I just kept "My" and will change it later if we find something nicer.

**Pull request merged**